opengever/maintenance/scripts/set_emm_ip_ranges.py

```diff
@@ -0,0 +1,54 @@
+from opengever.base.ip_range import valid_ip_range
+from opengever.maintenance.debughelpers import setup_app
+from opengever.maintenance.debughelpers import setup_option_parser
+from opengever.maintenance.debughelpers import setup_plone
+from opengever.officeconnector.interfaces import IOfficeConnectorSettings
+from plone.registry.interfaces import IRegistry
+from zope.component import getUtility
+from zope.interface import Invalid
+import sys
+import transaction
+
+
+def set_emm_ip_ranges(ip_ranges):
+    """Search all MS OneNote documents and fixes the contentType.
+    """
+    try:
+        valid_ip_range(ip_ranges)
+        registry = getUtility(IRegistry)
+        settings = registry.forInterface(IOfficeConnectorSettings)
+        settings.office_connector_disallowed_ip_ranges = ip_ranges
```

Comment:
you could set it easier with https://docs.plone.org/develop/plone.api/docs/portal.html#set-plone-app-registry-record